dist/common/scripts/scylla_util.py

```diff
@@ -518,6 +586,37 @@ def is_recommended_instance_size(self):
         if int(self.instance_size()) > 1:
             return True
         return False
+
+    # @staticmethod
+    # def get_file_size_by_seek(filename):
+    #     "Get the file size by seeking at end"
+    #     fd= os.open(filename, os.O_RDONLY)
+    #     try:
+    #         return os.lseek(fd, 0, os.SEEK_END)
+    #     finally:
+    #         os.close(fd)
+
```

Comment:
What's this comment?

Reply:
removing, it was just a placeholder for the refactoring